### DISCUSSION TITLE
Comfyui Workflows

### DISCUSSION THREAD

**Discussion by Mohsyn** [receyuki/stable-diffusion-prompt-reader#76] [General]
For files with Comfyui workflow, why is it not possible to just show raw data, as it is, and allow user to copy / edit / clear, themselves ?

**Comment by receyuki** [comment]
Since ComfyUI's data is super complicated (it contains the entire workflow and all the nodes used), it's tough for me to display it in a human readable format. So, the app won't show the raw data directly and you can't edit it in ComfyUI's format. But that doesn't mean you can't get it.

You can use either the Copy button below the UI or the Export button to get the complete original workflow data.

As for clearing metadata, not being able to clear metadata from ComfyUI images that can't be parsed is a bug, so thanks for pointing that out.
